src/styles/main.scss

```diff
@@ -1,3 +1,12 @@
+@import "./utils/variables";
+@import "./utils/mixins";
+@import "./blocks/calendar";
+
 body {
```

Comment:
It's better to add an additional wrapper and pass these rules for it instead of directly `body`. Not critical for this project but a good point for the future work.